manifests/tasks_service.pp

```diff
@@ -0,0 +1,28 @@
+# = foreman-tasks service
```

Comment:
nitpick about the name, imho dynflow or background-jobs makes sense not to confuse with task plugin.